docs/examples/machine_translation/transformer.md

```diff
@@ -243,248 +240,11 @@ print('The German translation is:')
 print(sample_tgt_seq)
 ```
 
-## Train Your Own Transformer
-
-In this subsection, we will go though the whole process about loading translation dataset in a more unified way, and create data sampler and loader, as well as define the Transformer model, finally writing training script to train the model yourself.
+## Next
```

Comment:
Since the Next here is a little bit duplicated with the Next button on the page, we can probably remove the section title.